src/scenes/ApplicationRouter/ApplicationsList.tsx

```diff
@@ -7,12 +7,15 @@ import { DateTime } from 'luxon';
 import ApplicationCard from 'src/components/Application/ApplicationCard';
 import PageHeader from 'src/components/PageHeader';
 import Button from 'src/components/common/button/Button';
+import { useOrganization } from 'src/providers';
 import strings from 'src/strings';
+import { isAdmin } from 'src/utils/organization';
 
 import { useApplicationData } from '../../providers/Application/Context';
 import NewApplicationModal from './NewApplicationModal';
 
 const ApplicationListView = () => {
+  const { selectedOrganization } = useOrganization();
   const { isTablet, isMobile } = useDeviceInfo();
   const { allApplications } = useApplicationData();
   const [isNewApplicationModalOpen, setIsNewApplicationModalOpen] = useState<boolean>(false);
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +25]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->We should check `isAdmin` in this line as well<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/3169?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->